modules/conversantBidAdapter.js

```diff
@@ -64,148 +176,21 @@ export const spec = {
     return true;
   },
 
-  /**
-   * Make a server request from the list of BidRequests.
-   *
-   * @param {BidRequest[]} validBidRequests - an array of bids
-   * @param bidderRequest
-   * @return {ServerRequest} Info describing the request to the server.
-   */
-  buildRequests: function(validBidRequests, bidderRequest) {
-    const page = (bidderRequest && bidderRequest.refererInfo) ? bidderRequest.refererInfo.page : '';
-    let siteId = '';
-    let pubcid = null;
-    let pubcidName = '_pubcid';
+  makeBidUrl: function(bid) {
```

Comment:
Maybe move `makeBidUrl` outside of `spec` because it's not a part of the interface.